client/my-sites/plugins/sidebar/index.tsx

```diff
@@ -0,0 +1,42 @@
+import clsx from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import GlobalSidebar from 'calypso/layout/global-sidebar';
+import SidebarItem from 'calypso/layout/sidebar/item';
+import SidebarMenu from 'calypso/layout/sidebar/menu';
+import { SidebarIconCalendar } from './icons';
+import './style.scss';
+
+interface Props {
+	path: string;
+	isCollapsed: boolean;
+}
+const PluginsSidebar = ( { path, isCollapsed }: Props ) => {
+	const translate = useTranslate();
+
+	return (
+		<GlobalSidebar
+			className={ clsx( 'sidebar--plugins', { 'is-collapsed': isCollapsed } ) }
+			siteTitle={ ! isCollapsed && translate( 'Plugins' ) }
+			requireBackLink={ true }
```

Comment:
Please, rebase your PRs before merging to trunk. Because #90499 landed before merging this PR, this line was introducing an ESLint error. I've resolved this in #90822.

Reply:
Sorry to hear that, but I actually rebased it a few hours before merging back to the trunk.

Reply:
Curious. Something might be off with the check, I'll investigate.

Reply:
Fix coming up in https://github.com/Automattic/wp-calypso/pull/90877. Sorry for the noise!